src/image/rotate_worker.cpp

```diff
@@ -16,11 +16,12 @@ void RotateWorker::Execute () {
     const float nangle = cimg::mod(_degs, 360.0f);
     if (cimg::mod(nangle, 90.0f) != 0) {
         CImg<unsigned char> * res;
+        int channels = _cimg->spectrum();
```

Comment:
Sorry for the unclean commit here, really apologise, not sure how that happened. I can revert, commit, push, merge, pull, then kant can merge and update or, this code is probably better, has no impact on hardcoding or not and definitely needs to come back later. What do you think?